Sources/PointFree/Episode/Episode.swift

```diff
@@ -5,44 +5,59 @@ public struct Episode {
   public private(set) var blurb: String
   public private(set) var codeSampleDirectory: String
   public private(set) var exercises: [Exercise]
+  public private(set) var fullVideo: Video?
   public private(set) var id: Id
   public private(set) var image: String
+  public private(set) var itunesImage: String?
   public private(set) var length: Int
   public private(set) var permission: Permission
   public private(set) var publishedAt: Date
   public private(set) var sequence: Int
   public private(set) var sourcesFull: [String]
   public private(set) var sourcesTrailer: [String]
   public private(set) var title: String
+  public private(set) var trailerVideo: Video?
   public private(set) var transcriptBlocks: [TranscriptBlock]
 
+  public struct Video {
+    public private(set) var bytesLength: Int // TODO: Tagged<Bytes, Int>?
```

Comment:
we have to send the byte length in the rss payload, this would be a good use for tagged someday